vcsclient/github_test.go

```diff
@@ -210,6 +210,23 @@ func TestGitHubClient_DownloadRepository(t *testing.T) {
 	assert.Error(t, err)
 }
 
+func TestGitHubClient_DownloadFileFromRepository(t *testing.T) {
+	ctx := context.Background()
+	downloadURL := "https://jfrog.com"
+	name := "hello-world"
+	client, cleanUp := createServerAndClient(t, vcsutils.GitHub, false, &[]github.RepositoryContent{{DownloadURL: &downloadURL, Name: &name}}, "/repos/jfrog/repo-1/contents/?ref=branch-1", createGitHubHandler)
+	defer cleanUp()
+	_, statusCode, err := client.DownloadFileFromRepo(ctx, owner, repo1, branch1, "hello-world")
+	assert.NoError(t, err)
+	assert.Equal(t, http.StatusOK, statusCode)
```

Comment:
We can also check that the response is not empty
```suggestion
	content, statusCode, err := client.DownloadFileFromRepo(ctx, owner, repo1, branch1, "hello-world")
	assert.NoError(t, err)
	assert.Equal(t, http.StatusOK, statusCode)
	assert.NotEmpty(t, content)
```

Reply:
done